pkgs/tools/misc/vimpager/build.nix

```diff
@@ -18,6 +19,10 @@ stdenv.mkDerivation {
     rev    = version;
   };
 
+  postPatch = lib.optionalString (version != "2.06") ''
```

Comment:
So 2.07 and 2.05 don't need this? Why do we need this condition at all?

Reply:
I'm open to changing this however you'd like to suggest, it's just a bit of a messy situation because...
- 2.06 was the last tagged release of vimpager from 2015, and is like 400 commits behind master
- `vimpager-latest` is a more up-to-date derivation using a semi-recent git commit
- build.nix is used as a common builder for both
- `version` can be either `2.06` or a git rev/hash

An alternative might be to add the patch in an `.overrideAttrs` in latest.nix instead?

Reply:
> An alternative might be to add the patch in an .overrideAttrs in latest.nix instead?

That sounds like a cleaner solution which I think we should choose here.

Reply:
moved to `latest.nix` in [last push](https://github.com/NixOS/nixpkgs/compare/3fce78251c941a6d94dcee703a21d003be66be88..fda25f88d238899ecdc2b195e00855db24b761b7)